package/docker/Dockerfile

```diff
@@ -5,4 +5,7 @@ COPY package/docker/scripts/start.sh start.sh
 RUN chmod +x start.sh
 RUN chmod +x wait-for.sh
 COPY feed-integration-webapp/target/crater-atomfeed.jar .
-CMD [ "./start.sh" ]
\ No newline at end of file
+ENV SERVER_OPTS="-Xms512m -Xmx1024m -XX:PermSize=256m -XX:MaxPermSize=512m"
```

Comment:
was there heap issues to warrant increase in the size?

Reply:
@N0-man i will check with @atish160384 and get back on this one. I copied along with debug option from older commits.

Reply:
@N0-man got to know from @atish160384 that SERVER_OPTS was required when the dev machine has low resource configuration. But I don't think it's required now. so removed SERVER_OPTS in the later commits. Please check